.docker-env-example

```diff
@@ -1,4 +1,4 @@
-POSTGRES_DB=postgres
-POSTGRES_NAME=postgres
-POSTGRES_USER=postgres
-POSTGRES_PASSWORD=postgres
\ No newline at end of file
+export POSTGRES_DB=postgres
+export POSTGRES_NAME=postgres
+export POSTGRES_USER=postgres
+export POSTGRES_PASSWORD=postgres
```

Comment:
I don't think this is correct. I don't think that docker will load this file correctly (it's used in the docker-compose file):
https://docs.docker.com/compose/how-tos/environment-variables/variable-interpolation/#env-file-syntax

Reply:
That's why we have them in two places in edrop:
https://github.com/diging/edrop-connector/blob/develop/.env_app_example#L17